app/services/invite_schools.rb

```diff
@@ -134,6 +134,75 @@ def send_induction_coordinator_sign_in_chasers
     end
   end
 
+  def send_induction_coordinator_choose_route_chasers
+    induction_coordinators_not_chosen_route = User.includes(:induction_coordinator_profile, schools: :school_cohorts)
+                                                  .joins(:induction_coordinator_profile)
+                                                  .left_outer_joins(schools: :school_cohorts)
+                                                  .where(school_cohorts: { id: nil })
+                                                  .where.not(last_sign_in_at: nil)
+    induction_coordinators_not_chosen_route.find_each do |induction_coordinator|
```

Comment:
Why `find_each` instead of `each`?

Reply:
Batch the queries. It's quite a beefy query, with thousands of results. Probably not necessary, but seemed prudent